metronome/checkpointing/models/src/io/iohk/metronome/checkpointing/models/Block.scala

```diff
@@ -55,7 +55,6 @@ object Block {
     val header = Header(
       parentHash = Block.Header.Hash(ByteVector.empty),
       postStateHash = Ledger.empty.hash,
-      bodyHash = body.hash,
       contentMerkleRoot = MerkleTree.Hash.empty
```

Comment:
Maybe the above is a better pattern, i.e. instead of `MerkleTree.Hash` having an `empty` value, it could be `MerkleTree.empty.hash`?

Reply:
Yes, I like that